docs/_guides/documents.md

```diff
@@ -141,14 +141,14 @@ If you fail to include the correct `_rev`, you will get the following sad error:
 Updating documents correctly
 -----------
 
-So to update Mittens' age, we will first need to fetch Mittens from the database, to ensure that we have the correct `_rev` before we put him back. We don't need to manually assign the `_rev` value here (like we did above), as it is already in the `doc` we're fetching. 
+So to update Mittens' age, we will first need to fetch Mittens from the database, to ensure that we have the correct `_rev` before we put them back. We don't need to manually assign the `_rev` value here (like we did above), as it is already in the `doc` we're fetching. 
 
 ```js
 // fetch mittens
 db.get('mittens').then(function (doc) {
-  // update his age
+  // update their age
```

Comment:
Dunno why I thought of "Mittens" as a boy-cat name. :) Funny what this tool reveals.